types/result.go

```diff
@@ -198,8 +199,8 @@ func (r TxResponse) String() string {
 		sb.WriteString(fmt.Sprintf("  Code: %d\n", r.Code))
 	}
 
-	if r.Data != nil {
-		sb.WriteString(fmt.Sprintf("  Data: %s\n", string(r.Data)))
+	if r.Data != "" {
+		sb.WriteString(fmt.Sprintf("  Data: %s\n", r.Data))
```

Comment:
can we use tab `\t` in this function? 
```suggestion
		sb.WriteString(fmt.Sprintf("\tData: %s\n", r.Data))
```

Reply:
oh no, please don't.... I'd even remove the trailing `\n`

Reply:
lol...the tab vs space war begins. idc really (go's standard is tabs), but I'd rather be consistent. What to do?

Reply:
plz no `\t` 🙏